vars/chunkyTemplate.groovy

```diff
@@ -9,7 +9,7 @@ def call(Map parameters = [:], body) {
 
     def chunkyImage = parameters.get('chunkyImage', 'fabric8/chunky-builder:0.0.2')
     def inheritFrom = parameters.get('inheritFrom', 'base')
-    def jnlpImage = (flow.isOpenShift()) ? 'fabric8/jenkins-slave-base-centos7:v54e55b7' : 'jenkinsci/jnlp-slave:2.62'
+    def jnlpImage = (flow.isOpenShift()) ? 'openshift/jenkins-slave-base-centos7:v4.0' : 'jenkinsci/jnlp-slave:2.62'
```

Comment:
@arilivigni @rupalibehera 
I was going to squash and merge the PR but then I remembered that we have added a change to the jnlp entrypoint to bail out as soon as master can't be reached. With this change won't we loose that? Any thoughts if we can get that change upstream first before changing the jnlp image? 

cc @kishansagathiya

Reply:
@sthaha I don't think this is a change we can merge upstream unless there is a clear requirement for it.  It would be better to do this outside the image if possible like I have done with the other changes.